targets/targets.json

```diff
@@ -9123,6 +9123,28 @@
             "network-default-interface-type": "WIFI"
         }
     },
+    "CY8CMOD_062S3_4343W": {
+        "inherits": ["MCU_PSOC6_M4"],
+        "features": ["BLE"],
+        "components_add": ["WHD", "4343W", "CYW43XXX"],
```

Comment:
What are these components? Are those drivers for external HW? If not they shouldn't be in components.

Reply:
Yes, these are drivers for the external Wifi and BT radio, which were changed to a component in #11840 .

Reply:
Also `CYW43XXX` ?

Reply:
Yes, all three are related to external radio support.